cmd/connectme/list/cli.go

```diff
@@ -0,0 +1,94 @@
+package list
+
+import (
+	"context"
+	"fmt"
+	"github.com/liggitt/tabwriter"
+	"github.com/pulumi/pulumi/sdk/v3/go/auto"
+	"github.com/pulumi/pulumi/sdk/v3/go/common/tokens"
+	"github.com/pulumi/pulumi/sdk/v3/go/common/workspace"
+	"github.com/spf13/cobra"
+	"os"
+	//"github.com/spf13/viper"
+)
+
+const (
+	tabwriterMinWidth = 6
+	tabwriterWidth    = 4
+	tabwriterPadding  = 3
+	tabwriterPadChar  = ' '
+	tabwriterFlags    = tabwriter.RememberWidths
+)
+
+var ()
+
+func Command() *cobra.Command {
+	command := &cobra.Command{
+		Use:   "list",
+		Short: "List all active connectme infrastructure",
+		Long:  `List all the connectme infrastructure from the backend store`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+
+			projectName := "connectme"
+			ctx := context.Background()
+
+			ws, err := auto.NewLocalWorkspace(ctx, auto.Project(workspace.Project{
+				Name:    tokens.PackageName(projectName),
+				Runtime: workspace.NewProjectRuntimeInfo("go", nil),
+			}))
+			if err != nil {
+				return fmt.Errorf("error listing connectme instances: %v", err)
+			}
+
+			stacks, err := ws.ListStacks(ctx)
+			if err != nil {
+				return fmt.Errorf("error listing connectme instances: %v", err)
+			}
+
+			writer := tabwriter.NewWriter(os.Stdout, tabwriterMinWidth, tabwriterWidth, tabwriterPadding, tabwriterPadChar, tabwriterFlags)
+			fmt.Fprintln(writer, "NAME\tLAST UPDATE\t# RESOURCES\tTYPE\tURL")
```

Comment:
I don't think we need `Resources` in the table because that number _should_ be the same between every stack and probably not super useful to the user. 

What would be super useful I think would be add what I am connected to, like the `subnetID` or something.

Reply:
Good point actually. I'll merge this as is an open an issue for fixing this

Reply:
https://github.com/jaxxstorm/connectme/issues/15